firecracker/rootfs/scripts/create_rootfs_image.sh

```diff
@@ -49,6 +51,21 @@ sudo chmod 777 "${MOUNT_POINT}"
 sudo debootstrap --include apt,nano "${DEBIAN_VERSION}" \
     "${MOUNT_POINT}"
 
+# Copy in the Plugin Bootstrap binary and associated systemd services
+(
+    ls /usr/local
+    # Destination must match what's in grapl-plugin-bootstrap-init.service
+    sudo cp "${PLUGIN_BOOTSTRAP_INIT_ARTIFACTS_DIR}"/plugin-bootstrap-init \
+        "${MOUNT_POINT}/usr/local/bin/init-grapl-plugin"
```

Comment:
Is there a reason why we're changing the name here? `plugin-bootstrap-init` -> `init-grapl-plugin`? Why not just keep it the same?

Reply:
good point. i was copying what was already in the .service file, bad move